src/Compilers/VisualBasic/Portable/Symbols/SubstitutedNamedType.vb

```diff
@@ -589,6 +590,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Next
         End Function
 
+        Friend Overrides ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
+            Get
+                Return OriginalDefinition.AllRequiredMembers
```

Comment:
>OriginalDefinition.AllRequiredMembers [](http://example.com/codeflow?start=23&length=37)

This doesn't look like the right thing to do. Shouldn't returned members be substituted?

Reply:
Including those from base, I guess

Reply:
BTW, what are we doing for C#?

Reply:
C# recalculates this on every type using `GetMembersUnordered`, so it should have the correct substitutions.